src/coreclr/jit/assertionprop.cpp

```diff
@@ -1332,72 +1332,72 @@ AssertionIndex Compiler::optCreateAssertion(GenTree*         op1,
                     op2Kind = O2K_CONST_DOUBLE;
                     goto CNS_COMMON;
 
-                CNS_COMMON:
-                {
-                    //
-                    // Must either be an OAK_EQUAL or an OAK_NOT_EQUAL assertion
-                    //
-                    if ((assertionKind != OAK_EQUAL) && (assertionKind != OAK_NOT_EQUAL))
-                    {
-                        goto DONE_ASSERTION; // Don't make an assertion
-                    }
+CNS_COMMON:
```

Comment:
One new things is labels are left-justified, and it appears that code blocks are then aligned to the preceding label. That's a bit weird.

Reply:
Looks like this setting: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#indentgotolabels

Reply:
I set `IndentGotoLabels: true` and this seems to be fixed.